config/locales/mailers/en.yml

```diff
@@ -190,6 +190,39 @@ en:
         html: "%{login}, the email associated with your account has been changed to %{email}"
         text: "%{login}, the email associated with your account has been changed to %{email}"
       subject: "[%{app_name}] Email changed"
+    claim_notification:
+      access:
+        html: Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please %{contact_support_link}.
+        text: Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please contact AO3 Support.
```

Comment:
The guidelines [here](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#general-guidelines) seem to suggest the link available in the HTML version should also be in the text version. Though I might be over interpreting there.

Reply:
I think I'll go with preserving the original behavior, unless AD&T wants to say something.